pihole-google.txt

```diff
@@ -7399,4 +7405,293 @@ www.tf-enterprise-devel.gcp.fitbit.com
+crashlyticsreports-pa.googleapis.com
+history.google.com
+android.googleapis.com
+people-pa.googleapis.com
+securetoken.googleapis.com
+www.googleapis.com
+chromesyncpasswords-pa.googleapis.com
+clientservices.googleapis.com
+cloudconfig.googleapis.com
+content-autofill.googleapis.com
+cryptauthdevicesync.googleapis.com
+cryptauthenrollment.googleapis.com
+datamixer-pa.googleapis.com
+deviceintegritytokens-pa.googleapis.com
+digitalassetlinks.googleapis.com
+firebaseremoteconfig.googleapis.com
+footprints-pa.googleapis.com
+geller-pa.googleapis.com
+gmscompliance-pa.googleapis.com
+growth-pa.googleapis.com
+infinitedata-pa.googleapis.com
+instantmessaging-pa.googleapis.com
+lamssettings-pa.googleapis.com
+mdh-pa.googleapis.com
+mobilemaps-pa-gz.googleapis.com
+mobilemaps-pa.googleapis.com
+myphonenumbers-pa.googleapis.com
+notifications-pa.googleapis.com
+performanceparameters.googleapis.com
+phonedeviceverification-pa.googleapis.com
+reminders-pa.googleapis.com
+safebrowsing.googleapis.com
+speechs3proto2-pa.googleapis.com
+ulp1p-pa.googleapis.com
+mtalk.google.com
+alt1-mtalk.google.com
+alt1.mobile-gtalk.l.google.com
+alt2-mtalk.google.com
+alt2.mobile-gtalk.l.google.com
+alt3-mtalk.google.com
+alt3.mobile-gtalk.l.google.com
+alt4-mtalk.google.com
+alt4.mobile-gtalk.l.google.com
+alt5-mtalk.google.com
+alt5.mobile-gtalk.l.google.com
+alt6-mtalk.google.com
+alt6.mobile-gtalk.l.google.com
+alt7-mtalk.google.com
+alt7.mobile-gtalk.l.google.com
+alt8-mtalk.google.com
+alt8.mobile-gtalk.l.google.com
+android.clients.google.com
+play.googleapis.com
+play-fe.googleapis.com
+playatoms-pa.googleapis.com
+playgateway-pa.googleapis.com
+userlocation.googleapis.com
+googleads.g.doubleclick.net
+static.doubleclick.net
+adx.g.doubleclick.net
+pubads.g.doubleclick.net
+securepubads.g.doubleclick.net
+static-doubleclick-net.l.google.com
+96d80cafb2cc6ad351ea76aea0f55d37.safeframe.googlesyndication.com
+ad.doubleclick.net
+googleads4.g.doubleclick.net
+dart.l.doubleclick.net
+cm.g.doubleclick.net
+pagead-googlehosted.l.google.com
+ssl-google-analytics.l.google.com
+adservice.google.com
+adservice.google.de
+pagead2.googleadservices.com
+pagead2.googlesyndication.com
+tpc.googlesyndication.com
+ade.googlesyndication.com
+www.googletagmanager.com
+www.googletagservices.com
+adssettings.google.com
+doubleclick.net 
+doubleclick.com 
+www.googleadservices.com
+chromefeedcontentsuggestions-pa.googleapis.com
+partnerad.l.doubleclick.net
+s-usc1c-nss-269.firebaseio.com
+firebaseremoteconfig.googleapis.com
+firebasestorage.googleapis.com
+firebaseinstallations.googleapis.com
+nmdp-b86eb-users-ko.firebaseio.com
+firebaselogging-pa.googleapis.com
+app-measurement.com 
+clients.l.google.com
+clients1.google.com
+clients2.google.com
+clients3.google.com
+clients4.google.com
+connectivitycheck.gstatic.com
+mobile-gtalk.l.google.com
+play-lh.googleusercontent.com
+sb.l.google.com
+supl.google.com
+redirector.gvt1.com
+beacons.gvt2.com
+beacons3.gvt2.com
```

Comment:
gvt.com domains are not linked to Google. These should be removed

Reply:
Should I delete all of them from gvt?

Reply:
Whoops, `gvt1.com` and `gvt2.com` are indeed Google's. Only `gvt.com` is not. It's probably best if you revert the last commit.

Reply:
como podria hacer eso ?. no me permite cargar las cosas.

Reply:
now I think they are already on